test/scale-infra/INSTRUCTIONS.md

```diff
@@ -0,0 +1,107 @@
+    ```
+
+    Use the variable `vpc_name` to specify the name for the [AWS virtual private cloud](https://docs.aws.amazon.com/vpc/latest/userguide/what-is-amazon-vpc.html) to deploy your infrastructure in a logically isolated virtual network: *We highly advise utilizing a unique name or the AWS IAM user ID as the identifier for your VPC*. Furthermore, use the variable `ec2_image` to specify the [Amazon machine image ID](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/AMIs.html), keeping in mind that it is region-specific.
+
+    
+    We advise utilizing the subsequent command to acquire the AMI ("ImageId") value for a specific region:
+
+    ```bash
+    aws ec2 describe-images --region <region_name> --filters "Name=architecture,Values=x86_64" "Name=name,Values=ubuntu/images/hvm-ssd/ubuntu-jammy-22.04-*"
+    ```
+
+    Upon completion of the script's execution, an Ansible inventory file containing the IP addresses of the master and worker nodes will be generated in the present directory at `.data/hosts_core`.
+
+2. Deploy a Kubernetes cluster:
+
+    ```bash
+    ansible-playbook -i .data/hosts_core deploy-masters.yaml --ssh-common-args='-o StrictHostKeyChecking=no'
+    ansible-playbook -i .data/hosts_core deploy-workers.yaml --ssh-common-args='-o StrictHostKeyChecking=no'
+    ```
+
+3. Deploy KubeStellar in the hosting cluster:
+
+    ```bash
+    ansible-playbook -i .data/hosts_core deploy_ks_core.yaml --ssh-common-args='-o StrictHostKeyChecking=no' -e 'ks_release=0.26.0'
+    ```
+
+    You can use the variable `ks_release` to specify the KubeStellar release. Kubestellar is deployed using the [KS helmchart](https://github.com/kubestellar/kubestellar/tree/release-0.26.0/core-chart) configured with a ITS of type host. 
+
+4. Create the WEC hosting instances:
+
+    ```bash
+    ansible-playbook create-ec2.yaml -e "cluster_name=wec region=us-east-2 vpc_name=<vpc_name> aws_key_name=mykey wecs_hosting_instances=1 instance_type=t2.xlarge archt=x86_64 ec2_image='<your-aws-ami>" 
+    ```
+
+    Use the variable `wecs_hosting_instances` to specify the number of ec2 instances to be created to host WEC kind clusters.
+    
+    Upon completion of the script's execution, an Ansible inventory file containing the IP addresses of the ec2 WEC hosting instances will be generated in the present directory at `.data/hosts_wec`.
+
+5. Create the WEC kind clusters:
+
+    a) Update the inventory of the WEC instances to include the IP address of a master node from the K8s cluster created in step 1 above:
+
+    First, find the required master's IP address info in the following file: `.data/hosts_core`.
+
+    Next, use the following command to see the content of the WEC ansible inventory:
+    ```bash
+    cat .data/hosts_wec
+    ```
+
+    Sample output: 
+    ```console
+    [masters]
+    <add master node info here!>
```

Comment:
In my testing, hosts_wec lists zero master nodes.

Reply:
You have to manually add the entry for the master node in the `hosts_wec` file. See the instructions at **step 5-a)**

Reply:
My point here is that at this point in the instructions is a display of what the reader should expect to see before doing that edit. The display is inaccurate and confusing. The imperative `<add master node info here!>` does not appear in that file at this time. A display of file contents does not logically contain an imperative. Slightly rewording would make the intent clearer, I think. How about making it "You will add master node info here"?

Reply:
I updated with the following content:

------------ updated context -----------------

Lastly, edit the contents of the file .data/hosts_wec to include the IP address and name of a master node in the [masters]  section, for example:

[masters]
master1 ansible_host=192.168.56.20

[add_workers]
worker1 ansible_host=192.168.56.1

-------------------------------------------------